src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -8152,6 +8173,8 @@ void ExportMusicXml::writeMeasure(const Measure* const m,
     if (isFirstActualMeasure) {
         writeStaffDetails(m_xml, part);
         writeInstrumentDetails(part->instrument(), m_score->style().styleB(Sid::concertPitch));
+    } else if (mpc.systemStart && !part->show()) {
+        writeStaffDetails(m_xml, part);
```

Comment:
What does this addition do, and why is it necessary?

Reply:
This belongs actually to another branch. Thanks for pointing it out.